### PR TITLE
Embed proper

### PR DESCRIPTION
**Pull request by Georgehe4**
Embedding features now fully integrated and functional.

### CODE REVIEW

**Comment by Georgehe4** [comment]
@lukechurch @devoncarew 
embed.html now runs as a function-less front end.

File: lib/embed/embed.dart [hunk 1]
```diff
@@ -0,0 +1,668 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dart_pad.mobile_ui;
```
Comment (devoncarew) [inline lib/embed/embed.dart]: nit: library name

**Comment by devoncarew** [comment]
Why is there both lib/embed.dart and lib/embed_temp.dart? `lib/embed.dart` doesn't seem to be referenced by html.

**Comment by devoncarew** [comment]
Can you add some comments to the either embed or embed temp that this copy of mobile/playground is provisional - we may re-write or merge it with the other files.

File: web/mobile_style.html [hunk 2]
```diff
@@ -28,6 +28,30 @@
     --paper-toggle-button-unchecked-ink-color: var(--text-primary-color);
   }
 
+  #errorToast {
```
Comment (devoncarew) [inline web/mobile_style.html]: did you mean to make these changes to mobile_style.html as part of the embedding PR? Are they addressing issues?

File: web/mobile_style.html [hunk 3]
```diff
@@ -28,6 +28,30 @@
...
+     bottom: 2px;
```
Comment (devoncarew) [inline web/mobile_style.html]: nit: (here and on line 40) 2 space indent

File: web/mobile_style.html [hunk 4]
```diff
@@ -28,6 +28,30 @@
...
+  html .height-max {
+    height:100%;
+  }
+  
+  paper-toggle-button {
+     padding-top: 10px;
+  }
+  
+  #rightContainer {
+    float:right;
```
Comment (devoncarew) [inline web/mobile_style.html]: nit: space after `:`

File: web/embed_temp.html [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+            </div>
+          </paper-header-panel>
+        </paper-drawer-panel>
+      </section>
+      </div>
+      <div style = "float:right; height:100%; width: 40%; position: relative;">
+      <section id="executable-section" flex>
+        <paper-header-panel class="fit">
+          <paper-toolbar>
+            <!--<paper-fab icon="arrow-back" class="back-button" mini></paper-fab>--.
```
Comment (devoncarew) [inline web/embed_temp.html]: the comment is not terminated correctly

File: web/embed_temp.html [hunk 6]
```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
```
Comment (devoncarew) [inline web/embed_temp.html]: Lets plan (in a follow-on PR) to split this file into three separate embeddings. Hopefully we can reduce code duplication by having some shared html and or js files.

**Comment by Georgehe4** [comment]
@devoncarew 
embed_temp will be removed in the near future. It is the version that was function and base off of Kasper's mobile changes.
The new embed is not hooked up to embed.dart yet since we need to re-do the functionality.

**Comment by devoncarew** [comment]
cool, lgtm. Let's merge and iterate on it.

**Pull request merged**